Rakefile

```diff
@@ -25,6 +25,36 @@ task(:push) do
   sh("bundle exec fastlane release")
 end
 
+task(:generate_team_table) do
+  require 'json'
+  content = ["<table>"]
+
+  contributors = JSON.parse(File.read("team.json"))
+  counter = 0
+  number_of_rows = 5
+
+  contributors.keys.shuffle.each do |github_user|
+    user_content = contributors[github_user]
+
+    content << "<tr>" if counter % number_of_rows == 0
```

Comment:
This looks really good on desktop/tablet but it's a bit small on mobile. Does anyone know enough html/css magic to make it look pretty on both? 😊

Reply:
What exactly is small? Could you post a screenshot? As it looks ok for me.

Reply:
e.g. here the rightmost 2 columns are cut off:

![screen shot 2018-01-11 at 11 37 44](https://user-images.githubusercontent.com/279407/34821414-e632ea3a-f6c3-11e7-8a07-3a44e95ecc35.png)

Reply:
If this was iOS I'd be using a `UICollectionView` but somehow I doubt html has a `<collection-view>` tag 😄

Reply:
It has if you want (meaning you can pun any string and it doesn't crash), but it won't have the functionality too. 😂  

I'm not sure how would this be achievable, as the HTML is obviously static and the DOM is constructed with multiple columns and I don't think you can change it with CSS. Except maybe if it uses flexbox :-?

Reply:
Yeah I did some quick testing, modifying the `generate_team_table` task like this: 

```ruby
task(:generate_team_table) do
  require 'json'
  content = ["<div id='team-table' style='display:flex; flex-wrap:wrap; flex-direction:row;' width='100%'>"]

  contributors = JSON.parse(File.read("team.json"))
  contributors.keys.shuffle.each do |github_user|
    user_content = contributors[github_user]

    content << "<div style='padding: 6px 13px; border: 1px solid #dfe2e5;'>"
    content << "<img src='https://github.com/#{github_user}.png?size=200' width=140>"
    content << "<h4 align='center'><a href='https://twitter.com/#{user_content['twitter']}'>#{user_content['name']}</a></h4>"
    # content << "<p align='center'>#{user_content['slogan']}</p>" if user_content['slogan'].to_s.length > 0

    content << "</div>"
  end
  content << "</div>"

  readme = File.read("README.md")
  readme.gsub!(%r{\<div id='team-table'.*\<\/div\>}m, content.join("\n"))
  File.write("README.md", readme)
  puts("All done")
end
```

Seems to make it work without hard-coding the row/column count, and just uses the space available

Reply:
Something like [this][1], though it should be tested on GitHub by actually committing it, because the preview adds extra elements which breaks this.

  [1]: http://jsbin.com/zusudar/edit?html,output

Reply:
Yep, in the "old days" we would have `float:left` on all those `div`s ;)

Reply:
does the github markdown allow div tags with inline styles?

Reply:
Ha, no, turns out you can't: https://github.com/jch/html-pipeline/blob/master/lib/html/pipeline/sanitization_filter.rb#L40

Boooooo

Reply:
have inline styles 👍 

how about using https://github.com/minimagick/minimagick and rendering a single image?, well thats drifting off a bit, don't think it should be too complicated...sorry

Reply:
Well, yes, the best solution is: rotate your phone 90 degrees 😆

Reply:
So anything we can do, or should just leave it and ignore mobile? On docs.fastlane.tools we can have it nicer anyway, also on fastlane.tools

Reply:
Yeah not much we can do I think, since GitHub is blocking everything